h2/src/main/org/h2/engine/Database.java

```diff
@@ -336,6 +336,12 @@ private void openDatabase(int traceLevelFile, int traceLevelSystemOut,
         }
     }
 
+    public long getLockTimeout() {
```

Comment:
why does this return long?

Reply:
I guess because it's milliseconds? But you are absolutely right, int is enough. I must have been smoking something at the time. :smile: